### PR TITLE
me do gramar wel

### LINKED ISSUE #445: typo on overwatch announcement
it says Attention - Your squad has been selected for Overwatch. Check your Status pane for objectives.
instead of Attention - Your squad has been selected for Overwatch. Check your Status panel for objectives.
the loss of this one l is ruining the game

### PR DESCRIPTION
**Pull request by F-OS**
fixes #445

### CODE REVIEW

**Comment by Supermichael777** [comment]
Pane is actually the proper term

A panel is "a particular arrangement of information grouped together for presentation to users in a window or pop-up"
https://en.wikipedia.org/wiki/Panel_(computer_software)

Pane: 2.(computing, graphical user interface) A layer in the build-up of a GUI.
https://en.wiktionary.org/wiki/pane#Noun

So you see because its not an independently movable screen element pane is the proper term.

**Comment by F-OS** [comment]
> tldr
> 
> panel sounds better.

**Comment by Supermichael777** [comment]
And thus literacy dies more

**Comment by Rohesie** [comment]
Doesn't seems to be a real issue, as Supermichael777 has noted.

**Comment by Qustinnus** [comment]
@Supermichael777 using language that noone uses and people assume is incorrect is bad

**Comment by Rohesie** [comment]
So this is how grammar dies… with thunderous applause.
If @Qustinnus wants to merge it, it's up to him.

**Comment by ninjanomnom** [comment]
Pane isn't obscure and the usage here is correct. There's no reason to change this.

**Comment by F-OS** [comment]
#445 needs closing too.
